### PR TITLE
moved ng-app directive up a level so it can be used multiple times

### PR DESCRIPTION
**Pull request by OwenMelbz**
Its a little “hacky” in the sense, there is currently no proper asset/javascript flow, which makes this a little more code than you’d traditionally use.

### CODE REVIEW

File: src/resources/views/fields/table.blade.php [hunk 1]
```diff
@@ -66,7 +66,7 @@
 
         </table>
 
-        <div class="array-controls btn-group">
+        <div class="array-controls btn-group m-t-10">
```
Comment (tabacitu) [inline src/resources/views/fields/table.blade.php]: Thank you! I was planning on doing that :-) It really bugged me.

**Comment by tabacitu** [comment]
Ugh... Yeah, I don't really like the fact that we now have the angular attribute on every create & edit forms, even though the fields themselves may not use angular, but... i guess they aren't doing any harm and the solution works... 

Thanks! I guess we should keep thinking, maybe we'll find a better solution in the future.

Cheers! Good job!

**Comment by OwenMelbz** [comment]
if eventually a proper javascript/css/asset flow gets implemented you can bootstrap the module/controller yourself something similarly too

```js
jQuery(document).ready(function($) {
    angular.bootstrap($('data-table'), ['backpackTableApp']);
});
```

then you dont need the extra `ng-app` definition, but yeah needs proper asset queue for it to work reliably

**Comment by tabacitu** [comment]
Hmm... then, can't we do something like that right now?
1. remove the ng-app from ```create.blade.php``` and ```edit.blade.php```;
2. in ```table.blade.php``` we define the ng-app on the ```form``` element, using jQuery;

That would encapsulate the ```table``` field into one file, ```table.blade.php```, right?

**Comment by OwenMelbz** [comment]
done done done -> https://github.com/Laravel-Backpack/CRUD/pull/154

**Pull request merged**